chassis-test-app/docker-compose.yaml

```
@@ -12,10 +12,30 @@ services:
     volumes:
       - mongo_data:/data/db
     healthcheck:
-      test: [ "CMD", "mongosh", "--username", "root", "--password", "secret", "--eval", "db.adminCommand('ping')" ]
+      test: [ "CMD", "mongosh", "--username", "local_username", "--password", "local_password", "--eval", "db.adminCommand('ping')" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)